opm/input/eclipse/share/keywords/900_OPM/D/DIFFMICP

```diff
@@ -8,7 +8,6 @@
       "keyword": "TABDIMS",
       "item": "NTPVT"
     },
-    "requires": ["MICP"],
     "items": [
```

Comment:
I would have been useful with a || logic for "requires" as well. But that is not for this PR. For now removing is okay.